package.json

```diff
@@ -36,22 +37,22 @@
     "stylelint:fix": "stylelint \"**/*.css\" --fix",
     "test": "echo 'Run `npm run test-all` to run `npm test` for every package'",
     "test-all": "npm run type-check && lerna run test --concurrency=2 --stream",
-    "test-e2e": "start-server-and-test start http://localhost:3333 'npm run cypress -- run --config-file ./cypress/cypress.json'",
+    "test-e2e": "start-server-and-test e2e-test-studio http://localhost:3355 'npm run cypress:run'",
     "test-perf": "start-server-and-test perf-studio http://localhost:3344 'ts-node -r dotenv/config perf/suite'",
     "updated": "lerna updated",
     "watch": "gulp watch",
     "release-notes": "node scripts/printReleaseNotesTemplate.js",
     "versions": "node scripts/normalizeDependencyVersions.js",
     "normalize-pkgfields": "node scripts/normalizePackageFields.js",
     "prettify": "lerna run clean && npm run prettier -- --write",
-    "prettier": "node_modules/.bin/prettier '**/*.{ts,js,tsx,jsx}'",
+    "prettier": "prettier '**/*.{ts,js,tsx,jsx}'",
     "lint-next": "npm run eslint -- `git diff --name-only next --diff-filter ACMRTUXB`",
     "verify-format": "lerna run clean && npm run prettier -- -c",
     "postinstall": "npm run bootstrap",
     "vercel-build": "npm run build && cd examples/test-studio && npm run build -- -y",
     "type-check": "tsc -b ./packages --force",
-    "cypress": "cypress",
-    "cypress:open": "cypress open --config-file cypress/cypress.json"
+    "cypress:open": "cypress open --config-file cypress/cypress.json",
+    "cypress:run": "cypress run --config-file cypress/cypress.json"
```

Comment:
Maybe we should use `-` instead of `:` to be consistent with the rest of the file? 🙂

Reply:
Sure :)

Reply:
On second though, we do it a few other scripts already.

Isn't it a good way to give scripts some modifiers? E.g.:

- `npm run lint-examples`
- `npm run lint-examples:fix`